code/game/machinery/computer/dna_console.dm

```diff
@@ -528,7 +528,7 @@
 	var/mob/living/carbon/viable_occupant = null
 	if(connected)
 		viable_occupant = connected.occupant
-		if(!istype(viable_occupant) || !viable_occupant.dna || (viable_occupant.disabilities & NOCLONE))
+		if(!istype(viable_occupant) || !viable_occupant.dna || (viable_occupant.disabilities & NOCLONE) || (viable_occupant.dna.species.id == "plasmaman"))
```

Comment:
NOCLONE is what husks use iirc. Making a species flag version might be a good idea, as it would also be needed for synths as cloning them breaks their disguises.